src/handlers.ts

```diff
@@ -611,6 +617,23 @@ export function createRepostActionHandler(accountService: AccountService) {
 
         const actor = await apCtx.getActor(ACTOR_DEFAULT_HANDLE); // TODO This should be the actor making the request
 
+        if (!actor?.id) {
+            ctx.get('logger').info('Invalid Repost - no actor id');
+            return;
+        }
+
+        if (!post.id) {
+            ctx.get('logger').info('Invalid Repost - no post id');
+            return;
+        }
+
+        const account = await accountService.getByApId(actor.id);
```

Comment:
I think we should use `accountService.getDefaultAccountForSite` (or however it's called) here, rather than depending on the `actor` - because eventually the `actor` won't be present in this method

Reply:
Done